.github/helm/affine-cloud/templates/stateful-set.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
```

Comment:
Let's create pg by the official postgres helm charts: https://artifacthub.io/packages/helm/bitnami/postgresql